package.json

```diff
@@ -55,7 +55,7 @@
     "yarn": "^1.16.0"
   },
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "start": "adb reverse tcp:8081 tcp:8081 && node node_modules/react-native/local-cli/cli.js start",
```

Comment:
Why this? metro takes care of this automatically

Reply:
Does it? it only worked for me with `run:android` but when I just use `start` I got a `Could not connect to development server` error...

Reply:
yeah I've had to run `adb reverse` almost every time